dataland-frontend/src/components/resources/frameworkDataSearch/FrameworkDataSearchFilters.vue

```diff
@@ -80,7 +78,7 @@ export default defineComponent({
       sectorFilter: ref(),
       countryFilter: ref(),
       frameworkFilter: ref(),
-      getKeycloakPromise: inject<() => Promise<Keycloak>>("getKeycloakPromise"),
```

Comment:
We no longer need this, as this is part of the apiClientProvider, correct?

Reply:
I don't see this in my code anymore. Removed?? :)

Reply:
Will check it on my side